### DISCUSSION TITLE
Need help in finding out the possible cause for a migration error when using bbs2gh extension

### DISCUSSION THREAD

**Discussion by sd-bb-mig** [github/gh-gei#1322] [Q&A] [answered]
Hi All,
We are working in migrating repositories from bitbucket server to Github Cloud and in one of the repository during migration, we are getting below error message
`[2024-12-10T16:13:22Z] ERROR -- Git source migration failed. Error message: A critical error occurred in the Rename Hidden Refs step of the git migration.`

Can you please help me find the possible cause for this error as I am needing help to track the issue whether it is in commits / PR's

**Comment by tambling** [comment, marked as answer]
Hi, we don't provide customer support in this repository. You can raise this to [GitHub Support](https://support.github.com) (ideally including your migration ID), and they will make sure it gets escalated to the right team.
